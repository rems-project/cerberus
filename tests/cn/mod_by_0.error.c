/* Fails because it doesn't require the second operand to be non-zero. */

int mod (int x, int y)
/*@  ensures return == x % y; @*/
{
    return x % y;
}