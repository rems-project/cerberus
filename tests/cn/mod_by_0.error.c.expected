return code: 1
[1/1]: mod -- fail
tests/cn/mod_by_0.error.c:6:12: error: Undefined behaviour
    return x % y;
           ~~^~~ 
the value of the second operand of a '%' operator is zero (ยง6.5.5#5, sentence 2)
State file: file:///tmp/state__mod_by_0.error.c__mod.html
